### ISSUE TITLE
Make dockerfile to load onto AWS Lambda

### ISSUE THREAD

**Issue by santolucito** [Barnard-PL-Labs/tsltools#14]
We need an API for tslsynth. @nupurd89 discovered we cannot just upload binaries because the default AWS Lambda image has too old a version of GCC. Instead, we will need to upload a dockerfile that specifies a newer distro of linux, which in turn has a newer version of GCC, so we can run ltlsynt (a dependency of tslsynth)

**Comment by santolucito** [comment]
can you add the dockerfile to this repo? I'd like to try it out as well and see if we can figure out the build issues.

**Comment by nupurd89** [comment]
Update (3/22): I am still running into issues while building the Dockerfile. I am running into the same error when running the line 'Run apt-get install spot' because it can't locate the package spot. I have tried to install spot in different ways as shown [here](https://spot.lrde.epita.fr/install.html), but none of them seem to work well or actually install spot. I uploaded the Dockerfile to the repo.
